.github/workflows/bazel.yaml

```diff
@@ -15,11 +15,26 @@ on:
         # JSON is needed because list is not supported:
         # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputsinput_idtype
         description: |
-          JSON-formatted array of folders to run 'bazel test' in
+          JSON-formatted array of folders to run 'bazel test' in.
           For example, '[".", "e2e/smoke"]'
         type: string
+      exclude:
+        description: |
+          JSON-formatted array of exclusions to the generated matrix of tests.
+
+          By default, we don't test bzlmod with Bazel 5 (not supported)
+          and we don't test non-linux with Bazel 5 to minimize macOS and Windows minutes
+          since they are billed at 10X and 2X respectively:
+          https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions#included-storage-and-minutes          
```

Comment:
We also only test Windows and macOS on main tho you can enable these tests on PR using the branch name